src/style.css

```diff
@@ -0,0 +1,84 @@
+/* No passed !!! */
+
+body {
+  margin: 0;
+  font-family: "Roboto", "sans-serif";
+}
+
+button {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;/* identical to box height */
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  margin-top: 20px;
+}
+
+button:hover {
+  color: #00acdc;
+  background: #fff;
+  box-sizing: border-box;

```

Comment:
don't set such styles for `:hover` state. It should be set for the default state. Even better to set if for all items on the page to prevent unexpected behavior on other blocks